execute/run_dag.sh

```diff
@@ -2,7 +2,7 @@
 
 snakemake \
 --use-conda \
---configfile config.yaml \
+--configfile configs/config.yaml \
```

Comment:
I don't think it matches run scripts from PAQR exactly, if I recall correctly the config was there passed as `$1`? But that's OK. As I wrote in the other PR - these things will be adjusted once I publish MAPP. This is just FYI.

Reply:
Yes, we only matched the directory structures and workflow related stuff. The run scripts are kind of an accessory...    
Great that you're gonna incorporate profiles in the future btw! :pray: